symplyphysics/laws/hydro/froude_number.py

```diff
@@ -0,0 +1,36 @@
+from sympy import Eq, solve, sqrt, S
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+                           validate_output, dimensionless, convert_to)
+
+
+# Description
+# Froude number characterizes the ratio between the force of inertia and
+# the external force, in the field of which the motion occurs, acting on
+# an elementary volume of liquid or gas.
+# Law: Fr = u / sqrt(g * L), where
+# u is velocity,
+# g is acceleration due to gravity,
+# L is a characteristic length.
```

Comment:
What's this?

Reply:
Not addressed. What's this length?

Reply:
What should I do? Provide further description for L?

Reply:
Yes. What is it?